dune_api_scripts/queries/parsed_app_data.sql

```diff
@@ -12,32 +13,36 @@ AS (
             app_data as app_id,
             referrer::json -> 'appCode' as app_code,
             referrer::json -> 'version' as app_version,
-            (referrer::json -> 'metadata')::json as metadata
+            -- Notice there are 2 different environment fields.
+            -- One of them being utilized more than the other
+            referrer::json -> 'environment' as backend_env,
+            (referrer::json -> 'metadata')::json -> 'environment' as meta_env,
+            ((referrer::json -> 'metadata')::json -> 'referrer')::json -> 'address' as referrer,
+            ((referrer::json -> 'metadata')::json -> 'referrer')::json -> 'version' as referral_version,
+            ((referrer::json -> 'metadata')::json -> 'quote')::json -> 'version' as quote_version,
+            ((referrer::json -> 'metadata')::json -> 'quote')::json -> 'slippageBips' as slippage
```

Comment:
Still nitpicking, but in my opinion it makes sense to leave the `Bips` in the column name as it gives context to what the actual value means. Otherwise, this could be interpreted as a percentage for example.